tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -93,7 +93,9 @@ public function dataFileAsserts(): iterable
 			yield from $this->gatherAssertTypes(__DIR__ . '/data/catch-without-variable.php');
 		}
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/mixed-typehint.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-2600.php');
+		if (PHP_VERSION_ID < 80000) {
+			yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-2600.php');
```

Comment:
Why it's no longer tested on PHP 8+? If there are different asserts, please create `bug-2600-php8.php` for that.

Reply:
Updated.